### PR TITLE
fix(lichess): fixed a bit of unthemed stuff, aswell added prettier lichess support and more

### PR DESCRIPTION
**Pull request by kingofcube**
## 🔧 What does this fix? 🔧

fixed unthemed feeds, buttons and colors on the homepage, aswell ingame
i also added support for prettier lichess 

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: scripts/userstyles.yml [hunk 1]
```diff
@@ -427,7 +427,7 @@ userstyles:
     readme:
       usage: "Switch to a default Lichess light/dark theme via **Username** > **Background** for the best experience!"
       app-link: "https://lichess.org/"
-      current-maintainers: [*coopw1]
+      current-maintainers: [*coopw1, *kingofcube]
```
Comment (isabelroses) [inline scripts/userstyles.yml:430]: @coopw1 would you apricate having @kingofcube working with you?

File: styles/lichess/catppuccin.user.css [hunk 2]
```diff
@@ -580,5 +622,43 @@
...
+    #dasher_app .selector button.active::before {
+    color: @mantle!important;
+    }
+    #dasher_app .selector button:hover {
+    background-color: darken(@green, 30%);
+    color: @mantle;
+    }
+    :root:has(body[data-theme="system"]) #topnav.hover section:hover > a, :root:has(body[data-theme="system"]) #topnav section:active > a, :root:has(body[data-theme="system"]) #topnav div a, :root:has(body[data-theme="system"]) #dasher_app .selector button, :root:has(body[data-theme="system"]) #dasher_app .head, :root:has(body[data-theme="system"]) #dasher_app .piece .no-square, :root:has(body[data-theme="system"]) #dasher_app .theme .list button, :root:has(body[data-theme="system"]) #dasher_app .links a, :root:has(body[data-theme="system"]) #dasher_app .links button, :root:has(body[data-theme="system"]) #dasher_app .subs .sub, :root:has(body[data-theme="system"]) .site-buttons .dropdown a, :root:has(body[data-theme="system"]) .site-buttons .dropdown button {  
+     color: @surface1
+    }
```
Comment (isabelroses) [inline styles/lichess/catppuccin.user.css:662]: all the additional ` :root:has(body[data-theme="system"])` etc, seems a bit redundant.

File: styles/lichess/catppuccin.user.css [hunk 3]
```diff
@@ -1,8 +1,8 @@
-/* ==UserStyle==
+;/* ==UserStyle==
```
Comment (isabelroses) [inline styles/lichess/catppuccin.user.css:1]:
```suggestion
/* ==UserStyle==
```

File: styles/lichess/catppuccin.user.css [hunk 4]
```diff
@@ -68,6 +69,18 @@
...
 
+    /* Prettier Lichess Support */
+    --primaryColor: @accent-color !important;
+    --secondaryColor: @peach !important;
+    --tertiaryColor: @red !important;
+    --backgroundColor: @base !important;
+    --textColor: @text !important;
+    --surfaceColor: @crust !Important;
+    --surfaceColorHover: @mantle !Important;
+    --defaultWhite: @subtext1 !Important;
```
Comment (isabelroses) [inline styles/lichess/catppuccin.user.css:80]: `!important` shouldn't be have a capital I

**Review by isabelroses — CHANGES_REQUESTED** [review]

File: scripts/userstyles.yml (hunk 1 above)
Comment (isabelroses) [inline scripts/userstyles.yml:430]: you also need to add your name to the contributes list at the top of the file, see the first step of https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md#userstylesyml

File: styles/lichess/catppuccin.user.css (hunk 2 above)
Comment (kingofcube) [reply styles/lichess/catppuccin.user.css:662]: yea, shouldve removed it

File: scripts/userstyles.yml (hunk 1 above)
Comment (kingofcube) [reply scripts/userstyles.yml:430]: mkay

File: scripts/userstyles.yml
Comment (isabelroses) [inline scripts/userstyles.yml:1]: Actually, can you revert the changes made here. Ideally this would be a separate PR.

Comment (kingofcube) [reply scripts/userstyles.yml:1]: okay???

**Comment by isabelroses** [comment]
Please also see the linters complaints.

**Comment by kingofcube** [comment]
> Please also see the linters complaints.

mkay

**Comment by kingofcube** [comment]
> Please also see the linters complaints.

the linters work now

**Comment by isabelroses** [comment]
> > Please also see the linters complaints.
> 
> the linters work now

But the entire diff is 2 lines changed, with none of them being css changes.

**Comment by kingofcube** [comment]
> > > Please also see the linters complaints.
> > 
> > 
> > the linters work now
> 
> But the entire diff is 2 lines changed, with none of them being css changes.

well u wanted them to work so here ya go

**Comment by isabelroses** [comment]
I'm not going to deal with this behaviour.

**Comment by kingofcube** [comment]
> I'm not going to deal with this behaviour.

:/
